machina/templates/machina/forum_conversation/topic_list.html

```diff
@@ -29,16 +29,7 @@
                 <a href="{% url 'forum_conversation:topic' topic.forum.slug topic.forum.pk topic.slug topic.pk %}" class="topic-name-link">{{ topic.subject }}</a>{% if topic.is_locked %}&nbsp;<i class="fas fa-times-circle locked-indicator" title="{% trans 'This topic is locked' %}"></i>{% endif %}
                 <div>
                   <div class="topic-created">
-                  {% if topic.poster %}
-                    {% url 'forum_member:profile' topic.poster_id as poster_url %}
-                    {% blocktrans trimmed with poster_url=poster_url username=topic.poster|forum_member_display_name creation_date=topic.created %}
-                      By: <a href="{{ poster_url }}">{{ username }}</a> on {{ creation_date }}
-                    {% endblocktrans %}
-                  {% else %}
-                    {% blocktrans trimmed with poster_username=topic.first_post.username creation_date=topic.created %}
-                      By: {{ poster_username }} on {{ creation_date }}
-                    {% endblocktrans %}
-                  {% endif %}
+                  {% include "partials/posted_by.html" with poster=topic.poster creation_date=topic.created username=topic.first_post.username %}
```

Comment:
This will result in a new SQL query for each topic because the `topic.first_post` is done systematically now.